dolphinscheduler-ui-next/src/views/projects/workflow/definition/components/timing-modal.tsx

```diff
@@ -188,8 +188,8 @@ export default defineComponent({
       () => props.row,
       () => {
         timingState.timingForm.startEndTime = [
-          new Date(props.row.startTime),
-          new Date(props.row.endTime)
+          props.row.startTime? new Date(props.row.startTime) : new Date(),
```

Comment:
You can use `props.row.crontab` to determine whether timeForm init is needed. Beause timeForm.crontab and other's needs to be initialized.

Reply:
No problem. I'll improve that.